client/components/Profile/ProfilePictureForm/ProfilePictureForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import VerticalAlignHelper from '../../shared/VerticalAlignHelper/VerticalAlignHelper'
+import UploadDropzone from '../../UploadDropzone/UploadDropzone'
+import styles from './ProfilePictureForm.scss'
+
+class ProfilePictureForm extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      showForm: false
+    }
+  }
+
+  saveFile (file) {
+    this.uploadImage(file)
+  }
+
+  async uploadImage (file) {
+    await this.props.updateProfilePicture(file)
+
+    this.setState({ showForm: false })
+  }
+
+  renderImageUpload (field) {
+    return (
+      <UploadDropzone
+        className={styles.inputImageUpload}
+        saveFile={this.saveFile.bind(this)}
+      />
+    )
+  }
+
+  renderForm () {
+    return (
+      <form
+        id="uploadProfilePictureForm"
+        className={styles.form} >
+        {this.renderImageUpload()}
+      </form>
+    )
+  }
+
+  renderProfilePicture () {
+    const { user } = this.props
+    return (
+      <div
+        className={styles.profilePicture}
+        onClick={() => this.setState({ showForm: true })}>
+        <div className={styles.profilePictureInner}>
+          <img
+            className={styles.image}
+            src={
+              user.profilePicture ||
+              require('../../../images/profile-image.jpg')
```

Comment:
Why not import that image in the top and use it here? 
something like: 
import profileImage from '../../../images/profile-image.jpg'

then use it :)